src/main/java/org/apache/sysml/runtime/instructions/cp/ScalarScalarArithmeticCPInstruction.java

```diff
@@ -57,7 +57,7 @@ public void processInstruction(ExecutionContext ec) throws DMLRuntimeException{
 			//pre-check (for robustness regarding too long strings)
 			String val1 = so1.getStringValue();
 			String val2 = so2.getStringValue();
-			StringObject.checkMaxStringLength(val1.length() + val2.length());
+			// StringObject.checkMaxStringLength(val1.length() + val2.length());
```

Comment:
Can you add a comment here about removing this just to make is easier to figure out later on?